docs/more-info/build-server-setup/tfs-build-vnext.md

```diff
@@ -0,0 +1,26 @@
+# Visual Studio Online (Build vNext) Setup
+## Basic Usage
+In [Visual Studio Online](https://www.visualstudio.com/) build vNext (the web based build system) you can add a build step as follows:
+
+* **Build Step:** Command Line
```

Comment:
That's old and needs to change. That refers to using GitVersion.exe as a standalone. That won't get past the detached head issues that the VSO build step does. I'll update the docs.